bower.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "gpii-binder",
+  "version": "0.0.0",
+  "homepage": "https://github.com/GPII/gpii-binder",
+  "authors": [
+    "Tony <tony@raisingthefloor.org>"
+  ],
+  "description": "A Fluid component library that creates live bindings between model data and views.",
+  "moduleType": [
+    "amd"
+  ],
+  "license": "See LICENSE file for details.",
```

Comment:
Ugh.  I cleaned that up months ago in various `package.json` files but have obviously missed it in the `bower.json` file I started with.

Anyway, fixed, and I will hunt down the variants at least for the code already under review.